service/collector/set.go

```diff
@@ -0,0 +1,58 @@
+package collector
+
+import (
+	"github.com/giantswarm/exporterkit/collector"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/google/go-github/github"
+)
+
+type SetConfig struct {
+	GithubClient *github.Client
+	Logger       micrologger.Logger
+}
+
+// Set is basically only a wrapper for the operator's collector implementations.
+// It eases the iniitialization and prevents some weird import mess so we do not
+// have to alias packages.
+type Set struct {
+	*collector.Set
+}
+
+func NewSet(config SetConfig) (*Set, error) {
```

Comment:
The usual collector set magic. Makes it super easy to extend with more collectors.